camus-sweeper/src/main/java/com/linkedin/camus/sweeper/CamusSweeper.java

```diff
@@ -216,16 +217,18 @@ public void run() throws Exception
       try
       {
         runCollectorForTopicDir(fs, topicName, new Path(topic.getPath(), sourceSubdir), destinationPath);
+        log.info("Shutting down priority executor");
+        executorService.shutdown();
```

Comment:
wait, this is [inside a loop](https://github.com/vinted/camus/pull/24/files#diff-34802ddc212164b4c2a5af563768e90bL209), is `shutdown` call intended?

> An ExecutorService can be shut down, which will cause it to reject new tasks. The shutdown() method will allow previously submitted tasks to execute before terminating